client/src/i18n/fr.js

```diff
@@ -2,6 +2,15 @@ module.exports = {
   general: {
     hello: 'Bonjour',
     yes: 'Oui',
-    no: 'Non'
+    no: 'Non',
+    login: 'Se connecter',
+    reset: 'Réinitialiser',
+    register: 'Registre',
```

Comment:
No "Registre" but "Inscription"

Reply:
Here will always be a problem for non french 👍  Thank you!